WebRoot/skins/_base/base4/skin.properties

```diff
@@ -101,8 +101,13 @@ LoginBannerPosition =   background-size: contain;
 LoginFooterColor    =   color: @AppC@;
 LoginCopyrightColor =   color: @hexToRGBA(AppC, 60)@;
 LoginInputField     =   width: 100%;  height: 40px;  @MediumRoundCorners@;  border: 1px solid @hexToRGBA(SecondaryBrandingColor, 50)@;  padding: 6px 10px;  @FontSize-big@
-
-InlineErrorPanel    =   color: @AppC@;  @FontSize-normal@
+ChangePasswordLoginSection        = top:40%;
+ChangePassWordLoginInfoPanel      = margin-bottom:@SkinWrapperPadding@;
+ChangePassWordLoginInfoPanelCell  = color:@AppC@; @FontSize-slightly-big@
+ChangePasswordLoginConfirmCell    = padding-bottom:0px;
+ChangePassWordLoginErrorMessage   = @ChangePassWordLoginInfoPanelCell@; padding-top:@SkinWrapperPadding@; padding-bottom:0px;
+ChangePasswordButton              = padding-top:30px;
```

Comment:
why have a padding for the button ? it would be generic if all the ". zLoginFieldContainer" elements including change password field has the padding-bottom. 
And then we won't need the line 107 here setting padding 0.